lona/server.py

```diff
@@ -1,10 +1,18 @@
+from typing import overload, Awaitable, Optional, TypeVar, Union
+from concurrent.futures import Future
 from functools import reduce
 import operator
 import logging
 import inspect
 import asyncio
+import sys
 import os
 
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
@maratori in the last pr we agreed on just using this import in all python versions. I would do the same here

Reply:
It was current pr :)
I pushed the changes just now.

Reply:
@maratori: Whoops :D thanks!

Reply:
np :)

Reply:
btw: is it necessary to mention you in each comment with @fscherf ?